deepreg/loss/image.py

```diff
@@ -63,7 +64,7 @@ def __init__(
         self,
         num_bins: int = 23,
         sigma_ratio: float = 0.5,
-        reduction: str = tf.keras.losses.Reduction.SUM,
+        reduction: str = tf.keras.losses.Reduction.NONE,
```

Comment:
will test this with experiment in https://github.com/YipengHu/DeepReg/tree/dev/experiments/mrus2